node/declaration.ts

```diff
@@ -30,19 +30,44 @@ export const getDeclarationsFromConfig = (
   // a function configuration object, we replace the path because that object
   // takes precedence.
   for (const declaration of [...tomlDeclarations, ...deployConfig.declarations]) {
-    const config = functionsConfig[declaration.function] ?? {}
+    const config = functionsConfig[declaration.function]
+
+    // If no config is found, add the declaration as is
+    if (!config) {
+      declarations.push(declaration)
+
+      // If we have path specified create a declaration for each path
+    } else if (config.path?.length) {
```

Comment:
I think `config.path` may be a string at this point? When that happens, do we want to be checking its `length`?

Reply:
Hmm I understand your rationale, but I don't think it's immediately obvious. Perhaps adding that context to the comment would help?

Reply:
Shall do!